lib/mongo_ecto/conversions.ex

```diff
@@ -7,68 +7,70 @@ defmodule Mongo.Ecto.Conversions do
     {:ok, dumped} = Ecto.Type.adapter_dump(Mongo.Ecto, type, value)
     dumped
   end
-  def to_ecto_pk(%{__struct__: _} = value, _pk),
-    do: value
+
+  def to_ecto_pk(%{__struct__: _} = value, _pk), do: value
+
   def to_ecto_pk(map, pk) when is_map(map) do
     Enum.into(map, %{}, fn
       {"_id", value} -> {Atom.to_string(pk), to_ecto_pk(value, pk)}
-      {key, value}   -> {key, to_ecto_pk(value, pk)}
+      {key, value} -> {key, to_ecto_pk(value, pk)}
     end)
   end
-  def to_ecto_pk(list, pk) when is_list(list),
-    do: Enum.map(list, &to_ecto_pk(&1, pk))
-  def to_ecto_pk(value, _pk),
-    do: value
 
-  def inject_params(doc, params, pk) when is_keyword(doc),
-    do: document(doc, params, pk)
+  def to_ecto_pk(list, pk) when is_list(list), do: Enum.map(list, &to_ecto_pk(&1, pk))
+  def to_ecto_pk(value, _pk), do: value
+
+  def inject_params(doc, params, pk) when is_keyword(doc), do: document(doc, params, pk)
+
   def inject_params(list, params, pk) when is_list(list),
     do: map(list, &inject_params(&1, params, pk))
-  def inject_params(%Ecto.Query.Tagged{tag: tag, type: type, value: {:^, _, [idx]} = value}, params, pk) do
+
+  def inject_params(
+        %Ecto.Query.Tagged{tag: tag, type: type, value: {:^, _, [idx]} = value},
```

Comment:
same here this should be destructured inside the function body

Reply:
Lets just keep it as is for this PR and we can have a conversation about it another time.